pintrest notification/src/Components/Notification.jsx

```diff
@@ -0,0 +1,33 @@
+const Notification = () => {
+
+    return <nav className="nav">
+      <>
+         <a href="https://www.pinterest.com/">
+          <div className="notify"> 
+          
+
+            <svg width="40px" height="40px" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Ideally, you want to move this outside of the component to a separate module and import it. Let me know if that makes sense.

Reply:
You mean put it in say, a component containing images and import it from there?